src/Sulu/Bundle/LocationBundle/Geolocator/GeolocatorLocation.php

```diff
@@ -68,6 +68,11 @@ class GeolocatorLocation
      */
     protected $latitude;
 
+    /**
+     * @var string
+     */
+    private $displayName;
```

Comment:
Shouldn't that be `displayTitle`?

Reply:
not sure there exist a `setDisplayName`: https://github.com/sulu/sulu/blob/e8600ca4e16275e67a2f9ff9432021c603a21c7c/src/Sulu/Bundle/LocationBundle/Geolocator/GeolocatorLocation.php#L167